src/ui/components/NetworkMonitor/RequestTable.tsx

```diff
@@ -78,73 +81,100 @@ const RequestTable = ({
     }
   };
 
+  const copyAsCurl = () => {
+    setContextMenuData(null);
+    navigator.clipboard.writeText("curl '" + contextMenuData.row.values.url + "'");
+
+    setShowCopied(true);
+    setTimeout(() => setShowCopied(false), 2700);
+    setTimeout(() => document.getElementById("showCopied").classList.remove("opacity-100"), 2000);
+    setTimeout(() => document.getElementById("showCopied").classList.add("opacity-0"), 2000);
```

Comment:
The way this is written (a chain of `setTimeout` calls that attempt to line up with a CSS transition) seems kind of brittle. If it's the effect you're going for, I don't have a better suggestion, but maybe we can make it a little more obvious what it's doing?

```suggestion
let timeout = setTimeout(() => {
  document.getElementById("showCopied").classList.remove("opacity-100");
  document.getElementById("showCopied").classList.add("opacity-0");
  
  // The fade out transition lasts 700ms
  timeout = setTimeout(() => {
    setShowCopied(false);
  }, 700);
}, 2000);

return () => {
  clearTimeout(timeout);
}
```

Also we should probably clear the timeout on unmount (as shown above) so we don't risk a potential undefined access error.

Reply:
Thanks! All good suggestions.